frontend/components/DataField.vue

```diff
@@ -48,36 +41,28 @@ export default {
       type: String,
       required: true,
     },
-    companyTags: {
-      type: Array,
-    },
-  },
-  methods: {
-    isMobile,
   },
 };
 </script>
 
 <style lang="pcss" scoped>
```

Comment:
I thought that we got rid of this pure css and use only TailwindCSS classes.

Reply:
We are going to replace all components with TailwindUI elements as described in pull request, because it's hard to recycle current templates.
They should already include Tailwind classes.